action.yml

```diff
@@ -17,92 +17,58 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: 'Checkout codebase'
-      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      with:
-        lfs: 'true'
-        submodules: 'true'
-        fetch-depth: 0
-        fetch-tags: true
+    - name: Checkout codebase
+      uses: ./mobile-android-pipelines/actions/checkout-codebase
 
     - name: Clean workspace
       uses: ./mobile-android-pipelines/actions/clean-workspace
 
-    - name: 'Detect architecture'
-      id: detect-arch
-      uses: ./mobile-android-pipelines/actions/detect-arch
```

Comment:
I never found a usage of this, so if you're not using it on the CRI the whole action can go

Reply:
Agreed, left it there just in case it was needed but will remove it